projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -20,6 +20,10 @@ Detecting Inconsistent Property Ranges Across Subclass Hierarchies Level 3 - 20
 
 1. Detect Classes Without Any Instances (Level 8 - 0 points)
 
+Title: Empty Class Detection
+Constraint Description: Identifies classes that have no instances associated with them.
+Severity: Warning
+
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 SELECT ?class
```

Comment:
most classes (all classes?) in CCO are of type owl:class 
I am not sure whether this would create issues. Are you getting results by running this on protege?